website/addons/wiki/templates/edit.mako

```diff
@@ -231,7 +232,7 @@
         <h3 class="modal-title">The permissions for this page have changed</h3>
       </div>
       <div class="modal-body">
-        <p>Your browser should refresh shortly&hellip;</p>
+        <p>Your browser should refresh shortly.&hellip;</p>
```

Comment:
This period should be removed. The entity `&hellip;` is a "horizontal ellipsis", so this results in `.…`.